automated install/basic-install.sh

```diff
@@ -109,14 +111,13 @@ c=$(( c < 70 ? 70 : c ))
 skipSpaceCheck=false
 reconfigure=false
 runUnattended=false
-INSTALL_WEB_SERVER=true
 # Check arguments for the undocumented flags
 for var in "$@"; do
     case "$var" in
         "--reconfigure" ) reconfigure=true;;
         "--i_do_not_follow_recommendations" ) skipSpaceCheck=true;;
         "--unattended" ) runUnattended=true;;
-        "--disable-install-webserver" ) INSTALL_WEB_SERVER=false;;
+        "--disable-install-webserver" ) INSTALL_WEB_SERVER=false INSTALL_WEB_INTERFACE=false;; 
```

Comment:
When the webserver install is skipped, this does not mean that the user does not want the web interface. It is possible that Apache or Nginx is used as webserver already and the user wants to continue is it. So if it is wanted to skip the dialog, a separate cmd option is required IMO.